src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/CSharpStatementBlockOnTypeFormattingTest.cs

```diff
@@ -15,30 +15,30 @@ public CSharpStatementBlockOnTypeFormattingTest(ITestOutputHelper output)
         }
 
         [Fact]
-        public async Task CloseCurly_IfBlock_SingleLine()
+        public async Task CloseCurly_IfBlock_SingleLineAsync()
```

Comment:
An analyzer was complaining about all of these tests not having an `Async` suffix so I updated the names.

Reply:
We should probably just disable that analyzer in the tests folder, but this is good too :)